lib/markbind/lib/parser.js

```diff
@@ -34,7 +35,16 @@ const BOILERPLATE_FOLDER_NAME = '_markbind/boilerplates';
  * Utils
  */
 
+/**
+* @throws Will throw an error if a non-absolute path or path outside the root is given
+*/
 function calculateNewBaseUrl(filePath, root, lookUp) {
+  if (!path.isAbsolute(filePath)) {
+    throw new Error(`Non-absolute path given to calculateNewBaseUrl: '${filePath}'`);
```

Comment:
Can you check if `throw` prints the stack trace?
Then we can just have "Non-absolute path given: '${filePath}'".

Reply:
It doesn't print the stack trace